### PR TITLE
Test gallery

### PR DESCRIPTION
**Pull request by domoritz**
- move to separate directory
- separate build
- compile vl and vg into one js file (no need for bower)

### CODE REVIEW

**Comment by domoritz** [comment]
Blocked by https://github.com/vega/vega/issues/647

**Comment by domoritz** [comment]
Rebased. Only needs a release of vega and then it's good to go.

**Comment by domoritz** [comment]
Ready

File: tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "typeRoots" : ["./typings", "node_modules/@types"]
   },
   "include": [
-    "**/*.ts",
+    "src/**/*.ts",
```
Comment (kanitw) [inline tsconfig.json]: Why this change?  This makes test fail to run.

Comment (kanitw) [inline tsconfig.json]: I guess you want to avoid test-gallery?  If so, add another line for test.

Comment (domoritz) [inline tsconfig.json]: Fixed
